src/WebJobs.Script.WebHost/Diagnostics/WebHostMetricsLogger.cs

```diff
@@ -15,6 +15,7 @@ public void BeginEvent(MetricEvent metricEvent)
             if (startedEvent != null)
```

Comment:
@davidebbo We also emit ETW events in Kudu. David might have more insights into that and how we might be able to follow the same model here (without unsafe blocks)